src/events/index.spec.js

```diff
@@ -614,4 +614,141 @@ describe('Events', function() {
       });
     });
   });
+
+  describe('subscribeEvent', function() {
+    context('when all the required parameters are provided', function() {
+      let expectedSubscription;
+      let promise;
+      let request;
+      let event;
+      let user;
+      let subscriptionFromServer;
+      let toCamelCase;
+
+      beforeEach(function() {
+        event = fixture.build('event');
+        user = fixture.build('contxtUser');
+
+        expectedSubscription = fixture.build('userEventSubscription');
+        subscriptionFromServer = fixture.build(
+          'userEventSubscription',
+          expectedSubscription,
+          {
+            fromServer: true
+          }
+        );
+
+        request = {
+          ...baseRequest,
+          post: sinon.stub().resolves(subscriptionFromServer)
+        };
+        toCamelCase = sinon
+          .stub(objectUtils, 'toCamelCase')
+          .callsFake((app) => expectedSubscription);
+
+        const events = new Events(baseSdk, request);
+        events._baseUrl = expectedHost;
+
+        promise = events.subscribeEvent(user.id, event.id);
+      });
+
+      it('posts the user event to the server', function() {
+        expect(request.post).to.be.calledWith(
+          `${expectedHost}/users/${user.id}/events/${event.id}`
+        );
+      });
+
+      it('formats the returned user event', function() {
+        return promise.then(() => {
+          expect(toCamelCase).to.be.calledWith(subscriptionFromServer);
+        });
+      });
+
+      it('returns a fulfilled promise with the new user event subscription', function() {
+        return expect(promise).to.be.fulfilled.and.to.eventually.equal(
+          expectedSubscription
+        );
+      });
+    });
+
+    context('when the user ID is not provided', function() {
+      it('throws an error', function() {
+        const event = fixture.build('userEventSubscription');
```

Comment:
You have generated a user event subscription but are calling it an event. Please be cognizant of the differences between events, users, and user event subscriptions

(You probably want to generate an event)